aimmo-game-creator/worker_manager.py

```diff
@@ -171,7 +171,7 @@ def __init__(self, *args, **kwargs):
         super(KubernetesWorkerManager, self).__init__(*args, **kwargs)
 
     def _create_game_rc(self, id, environment_variables):
-        environment_variables['SOCKETIO_RESOURCE'] = "game/%s/socket.io" % id
```

Comment:
Oh, I see.  I will read a bit about how this works. 

The reason why we wanted to change "game/%s/socket.io" to "/socket.io" is because we couldn't find out how to make it work like that from Unity.